### PR TITLE
Deploy services on server on merge on main branch

### PR DESCRIPTION
**Pull request by martinratinaud**
Also had to add in the secrets

- ANSIBLE_VAULT_PASSWORD located in shared password database
- KNOWN_HOSTS generated with `ssh-keyscan -H 51.75.169.235` on the server machine
- SSH_KEY for which I used `ota_org_github_actions` key on server

### CODE REVIEW

**Comment by martinratinaud** [comment]
Fixes https://github.com/ambanum/OpenTermsArchive/issues/716

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        with:
+          key: ${{ secrets.SSH_KEY }}
+          known_hosts: ${{ secrets.KNOWN_HOSTS }}
+      - run: echo "${{ secrets.ANSIBLE_VAULT_PASSWORD }}" > vault.key
+      - run: |
+          git config --global user.name 'OTA Bot'
+          git config --global user.email 'bot@opentermsarchive.org'
+      - run: pip install --upgrade setuptools
+      - run: pip install ansible==2.9.11
+      - run: ansible-playbook ops/site.yml -i ops/inventories/production.yml
```
Comment (Ndpnt) [inline .github/workflows/deploy.yml:26]: `ansible-playbook ops/app.yml -i ops/inventories/production.yml -t update-declarations`
But before add modify ops to specify this tag

Comment (Ndpnt) [reply .github/workflows/deploy.yml:26]: See #712

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        with:
+          key: ${{ secrets.SSH_KEY }}
+          known_hosts: ${{ secrets.KNOWN_HOSTS }}
+      - run: echo "${{ secrets.ANSIBLE_VAULT_PASSWORD }}" > vault.key
+      - run: |
+          git config --global user.name 'OTA Bot'
+          git config --global user.email 'bot@opentermsarchive.org'
+      - run: pip install --upgrade setuptools
+      - run: pip install ansible==2.9.11
+      - run: ansible-playbook ops/site.yml -i ops/inventories/production.yml -t update-declarations
```
Comment (Ndpnt) [inline .github/workflows/deploy.yml:26]: `ops/site.yml` --> `ops/app.yml` Only the instructions for the application, no need to go through all the infrastructure instructions.

Comment (martinratinaud) [reply .github/workflows/deploy.yml]: Done

**Review by Ndpnt — APPROVED** [review]

**Pull request merged**